static/dbstructure.config.php

```diff
@@ -1257,7 +1257,7 @@
 			"mimetype" => ["type" => "varchar(60)", "comment" => ""],
 			"height" => ["type" => "smallint unsigned", "comment" => "Height of the media"],
 			"width" => ["type" => "smallint unsigned", "comment" => "Width of the media"],
-			"size" => ["type" => "mediumint unsigned", "comment" => "Media size"],
+			"size" => ["type" => "bigint unsigned", "comment" => "Media size"],
```

Comment:
We don't need 18 trillions. `int` is more than enough.

Reply:
No, `int` is clearly not enough for e.g. video files in Full-HD or 4k which are WAY larger than what `int` can hold.

Reply:
`mediumint unsigned` is 4 GB maximum, which is too small for video file byte sizes.

Reply:
`int` is enough, have a look here: https://dev.mysql.com/doc/refman/8.0/en/integer-types.html

Reply:
I made a mistake in my previous post, here's the maximum for each size and their corresponding size in bytes:
- `mediumint unsigned`: 16777215 so 16 MB max
- `int unsigned`: 4294967295 so 4GB max (not unusual for video files)
- `bigint unsigned`: 2^64-1 so ~8 zetta bytes which is an acceptable maximum for video file sizes.
 
`bigint` is the correct way forward to avoid these field size errors.